packages/react/src/badge/Badge.css.ts

```diff
@@ -24,6 +24,7 @@ export const badge = recipe({
       rounded: "full",
     },
     style({
```

Comment:
Was this a requirement from designer? We talked about it last time with badge shapes and decided that we were fine with thin/thick badges because the content is always dynamic - and can be number/string etc